Базовая стилизация всех страниц

father-fury opened the pull request:
Добавил шрифты
Базовые стили элементам страниц
Изменил несколько названий изображений.

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/1790495/tasks/6)

:boom: https://htmlacademy-htmlcss.github.io/1790495-sedona-34/6/

father-fury committed:
Добавил базовую стилизацию

Добавил шрифты
Базовые стили элементам страниц
Изменил несколько названий изображений.

father-fury committed:
доработка симантики базовых стилей

изменение хронологии css блоков
удаление лишнего кода

gkarev commented:
Необходимо проверить названия шрифтов, и исправить, также не понятно, зачем дублируются стили подключения.
![image](https://user-images.githubusercontent.com/13588373/172709934-5f03cb17-1192-4732-b674-c306b9b8b14f.png)


Свойство `font-family` в селекторах необходимо указывать в формате(указан Arial):
`font-family: "PT Sans", Arial, sans-serif;`

Параметры шрифтов, необходимо указывать непосредственно элементу внутри которого находится текст, а не его родителю, что бы избегать лишнего наследования.
![image](https://user-images.githubusercontent.com/13588373/172710309-5f9d6007-ce68-4101-bf7b-ae3d9ca4b811.png)

father-fury committed:
Внес изменения по последним замечаниям

Изменил название пути подключения шрифтов.
Избавился от лишнего родительского кода и параметрами шрифтов.

gkarev commented:
Не находит шрифты, необходимо ещё раз внимательно проверить указанный путь
![image](https://user-images.githubusercontent.com/13588373/172932591-520036f9-f832-49cf-b807-be7af24c61ca.png)

father-fury committed:
изменил путь шрифтов

ошибку с vector.svg не совсем понял, изменил регистр заглавной буквы.
Ошибку про divicon.ico - вообще не понял.

father-fury commented:
изображение  vector.svg  отображается корректно, хотя был не верный регистр первой буквы в названии, я его изменил.
